utils/data_processing.py

```diff
@@ -0,0 +1,45 @@
+from Bio import SeqIO
+import pandas as pd
+
+
+def get_ids_without_version(id_x):
+    split = id_x.split('.')
+    if len(split) > 1:
+        return split[0]
+    else:
+        return id_x
+
+def parse_biomart_fasta(fasta_path):
```

Comment:
Add comment what this function does and why? What is format of input? What is format of output?

Reply:
done